promptsource/templates/ai2_arc/ARC-Challenge/templates.yaml

```diff
@@ -3,21 +3,17 @@ subset: ARC-Challenge
 templates:
   32f7eb4d-dd38-4503-b67d-a8a96ab40449: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: '{{choices.text | join("|||")}}'
```

Comment:
For this one (pick_false_options), we shouldn't set answer choices this way, because the possible outputs are lists of 3 choices. So we could make it some more complicated expression that generates all sequences of choices with one missing, or just leave it open ended since it's not an original task anyway.

Reply:
👍
I think we can leave it open-ended since it is not an original task